### PR TITLE
Fix bug in implementation of Ord, Eq for Pt and Mm

### PR DESCRIPTION
**Pull request by JulienSchminke**
Hello,

To Introduce, I am Julien and this is my first contribution on an open source project.

Firstly, I fixed this Issue https://github.com/fschutt/printpdf/issues/137. I changed the if statement so that it doesn't use the is_normal() function. Instead I check the output of the classify() function so that also zero numbers will accepted.

Secondly, I created a macro "impl_ord!" that offers the functionalities of the ord trait. I would like to add that because I am trying to implement a table method for this library and I need for that a max and a min function for the Mm type. 

Thirdly. I implemented Eq for Mm and Pt. This was required for the impl_ord Macro

I already tested it with unit tests and it works.

I would  really appreciate if you would merge it to the master branch so that everyone could benefit from the changes.

If you have any tips or ideas for me how I can contribute and help, please let me know.

Julien

### CODE REVIEW

File: Contributors.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+Julien Schminke
```
Comment (fschutt) [inline Contributors.md]: Add this to the Cargo.toml instead or the LICENSE.txt

**Review by fschutt — CHANGES_REQUESTED** [review]
Thank you, Julien (you don't have to be so formal online).

If you want to be added as a contributor, a better option is to either put it into the Cargo.toml file or in the LICENSE.txt.

**Comment by JulienSchminke** [comment]
Hey, thanks for your quick response. Where shall I put my name (and maybe email) in the Cargo.toml?

Maybe at authors?

**Comment by fschutt** [comment]
I would not like to have unnecessary dependencies here, also, please no tables in this crate, this stuff should go into [genpdf](https://docs.rs/genpdf/latest/genpdf/).

Please only one PR per problem, not multiple things into one PR.

**Comment by JulienSchminke** [comment]
What do you mean with PR?

Oh good to know. The table stuff should't  go to your branch (also if you would accept it). I wanted to push it only on my branch.

**Comment by JulienSchminke** [comment]
Do you maybe mean "push request" (PR). I only want to push the Fixed Bug commit. The rest shouldn't be pushed.
I need to know where or how to put me as a author of this crate in the Cargo.toml file and maybe how to hide the other commits with the table from this push request.

**Comment by fschutt** [comment]
> I need to know where or how to put me as a author of this crate in the Cargo.toml file and maybe how to hide the other commits with the table from this push request.

Just delete the lines of code and then git commit your changes. Put yourself in the `authors = [...]` array.

**Comment by JulienSchminke** [comment]
Thanks for your quick response. I've reverted the table changes and added me as an author.

File: src/table.rs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/*extern crate grid;
```
Comment (fschutt) [inline src/table.rs]: Please remove this file.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,30 +0,0 @@
-# Contribution and License
```
Comment (fschutt) [inline CONTRIBUTING.md]: Please keep this file.

**Review by fschutt — CHANGES_REQUESTED** [review]

**Pull request merged**